custom.props

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Version">
-    <VersionMajor>3</VersionMajor>
-    <VersionMinor>0</VersionMinor>
+    <VersionMajor>1</VersionMajor>
```

Comment:
Recommend to add comment here to specify that 1.1.0 refers to the ObjectModel release version and to keep track of the Renderer version (3.0.2) and/or to open bug for making one of the packages to use another way for versioning

Reply:
I'll address this when I update the version for the upcoming UWP rendering package :)